fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1028,7 +1028,7 @@ public void setMaxJoinNumBushyTree(int maxJoinNumBushyTree) {
     public boolean enableCTEMaterialize = true;
 
     @VariableMgr.VarAttr(name = IGNORE_COMPLEX_TYPE_COLUMN)
-    public boolean ignoreColumnWithComplexType = false;
+    public boolean ignoreColumnWithComplexType = true;
```

Comment:
change the session variable name to `enable_analyze_complex_type_column` and set default value to `false`